content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -562,6 +565,8 @@ in this specification must be under the `m.key.verification` namespace
 and any other event types must be namespaced according to the Java
 package naming convention.
 
+{{% event event="m.room.message$m.key.verification.request" %}}
```

Comment:
this presents itself as literally `m.room.message$m.key.verification.request`, which is obscure to say the least:

![image](https://user-images.githubusercontent.com/1389908/195170557-ffc0b232-7672-4262-b07d-f79c4592dc4d.png)

It's also in conflict to the way that `m.text` and co are represented, though I'm not sure that format would be terribly useful either:

![image](https://user-images.githubusercontent.com/1389908/195170915-27bae3f5-6690-4aea-91b5-31279b9fdc59.png)

Maybe the `render-event.html` template thing needs to be updated to honour the `title` of the event definition, then we can set that to something like "`m.room.message` with `msgtype: m.key.verification.request`" ?

Reply:
This and https://github.com/matrix-org/matrix-spec/pull/1271#discussion_r992649121 will be fixed by https://github.com/matrix-org/matrix-spec/pull/1283